recipes/kserve_images.rb

```diff
@@ -32,7 +32,7 @@
   EOH
 end
 
-kserve_image_names = ["kserve/kserve-controller", "kserve/agent", "kserve/storage-initializer"]
+kserve_image_names = ["kserve/kserve-controller", "kserve/agent", "kserve/storage-initializer", "kserve/alibi-explainer", "kserve/art-explainer"]
```

Comment:
Are `"kserve/alibi-explainer", "kserve/art-explainer"` required? We don't officially support explainers in Hopsworks, so they won't be referenced in the inferenceService yaml files, and will never be used.